third_party/move/move-vm/types/src/values/values_impl.rs

```diff
@@ -244,6 +276,8 @@ impl Container {
             Self::VecU256(r) => Rc::strong_count(r),
             Self::VecBool(r) => Rc::strong_count(r),
             Self::VecAddress(r) => Rc::strong_count(r),
+
+            Self::Locals(r) => Rc::strong_count(r),
```

Comment:
ditto

Reply:
these existing matches look pretty silly, if you agree do you mind making a macro that just applies the same method to all the variant internals (or even more for defining methods like len and rc_count)? at least two cases that we see here immediately.

Reply:
Good point, will do that!

Reply:
Hm, actually ended up not that easy, as we have vectors of different types, and no trait bounds, so injecting macro unifomrly for both len, rc_count and other places is a bit tricky. I will start a PR on top to clean this up